app/lib/services/admin/users_search.rb

```diff
@@ -25,6 +25,6 @@ def call
 private
 
   def default_scope
-    User.left_joins(:applications)
+    User.left_joins(:applications).order(email: :asc)
```

Comment:
This change is out of scope, but for some reason the intermittent failure on the admin specs had become persistent and I had to get this and the specs both ordering in the same way to guarantee proper testing. This'll also make it so the interface is consistent for admin users.